modules/videoio/src/cap_obsensor_capture.cpp

```diff
@@ -34,8 +34,10 @@ VideoCapture_obsensor::VideoCapture_obsensor(int index) : isOpened_(false)
 {
     static const obsensor::StreamProfile colorProfile = { 640, 480, 30, obsensor::FRAME_FORMAT_MJPG };
     static const obsensor::StreamProfile depthProfile = {640, 480, 30, obsensor::FRAME_FORMAT_Y16};
-    static const obsensor::StreamProfile gemini2depthProfile = {1280, 800, 30, obsensor::FRAME_FORMAT_Y14};
-    static const obsensor::StreamProfile astra2depthProfile = {640, 480, 30, obsensor::FRAME_FORMAT_Y14};
+    static const obsensor::StreamProfile gemini2DepthProfile = {1280, 800, 30, obsensor::FRAME_FORMAT_Y14};
+    static const obsensor::StreamProfile astra2DepthProfile = {640, 480, 30, obsensor::FRAME_FORMAT_Y14};
+    static const obsensor::StreamProfile megaColorProfile = {1280, 720, 30, obsensor::FRAME_FORMAT_MJPG};
```

Comment:
Why Motion JPEG? I propose to preserve the best quality by default, if possible.

Reply:
For platform compatibility reasons, the YUYV format does not pass on some of the lower performance platforms we tested.

Reply:
I propose to make MJPEG optional. (M)JPEG makes frames a bit blury. It affects feature detectors.

Reply:
MJPEG helps to save bandwidth (and connect multiple cameras).
On other cameras some formats+fps are not available without compression (especially FHD+) - MJPEG or H264.

BTW, MJPG is already used before this patch, see `colorProfile` on the line 35.